docs/source/deep-learning/pytorch/quickstart/pytorch_quickstart.ipynb

```diff
@@ -0,0 +1,1361 @@
+     "model_module": "@jupyter-widgets/controls",
+     "model_name": "DescriptionStyleModel",
+     "model_module_version": "1.5.0",
+     "state": {
+      "_model_module": "@jupyter-widgets/controls",
+      "_model_module_version": "1.5.0",
+      "_model_name": "DescriptionStyleModel",
+      "_view_count": null,
+      "_view_module": "@jupyter-widgets/base",
+      "_view_module_version": "1.2.0",
+      "_view_name": "StyleView",
+      "description_width": ""
+     }
+    }
+   }
+  }
+ },
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Quickstart with MLflow PyTorch Flavor\n",
+    "\n",
+    "In this quickstart guide, we will walk you through how to log your PyTorch experiments to MLflow. After reading this quickstart, you will learn the basics of logging PyTorch experiments to MLflow, and how to view the experiment results in MLflow UI.\n",
+    "\n",
+    "This quickstart guide is compatible with cloud-based notebook such as Google Colab and Databricks notebook, you can also run it locally."
+   ],
+   "metadata": {
+    "id": "S9KwXx6Hz3yl"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Install Required Packages"
+   ],
+   "metadata": {
+    "id": "CZwvVcIg0Z1P"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "id": "Sds9DYr9zcs0"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -q mlflow torchmetrics torchinfo"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "import mlflow\n",
+    "import torch\n",
+    "\n",
+    "from torch import nn\n",
+    "from torch.utils.data import DataLoader\n",
+    "from torchinfo import summary\n",
+    "from torchmetrics import Accuracy\n",
+    "from torchvision import datasets\n",
+    "from torchvision.transforms import ToTensor"
+   ],
+   "metadata": {
+    "id": "VH4jrEpRzkg-"
+   },
+   "execution_count": 2,
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Task Overview\n",
+    "\n",
+    "In this guide, we will demonstrate the functionality of MLflow with PyTorch through a simple MNIST image classification task. We will build a convolutional neural network as the image classifier, and log the following information to mlflow:\n",
+    "\n",
+    "- **Training Metrics**: training loss and accuracy.\n",
+    "- **Evalluation Metrics**: evaluation loss and accuracy.\n",
+    "- **Training Configs**: learning rate, batch size, etc.\n",
+    "- **Model Information**: model structure.\n",
+    "- **Saved Model**: model instance after training.\n",
+    "\n",
+    "Now let's dive into the details!"
+   ],
+   "metadata": {
+    "id": "3ucY0NjP0tPD"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Prepare the Data\n",
+    "\n",
+    "Let's load our training data `FashionMNIST` from `torchvision`, which has already been preprocessed into scale [0, 1). We then wrap the dataset into an instance of `torch.utils.data.Dataloader`."
```

Comment:
```suggestion
    "Let's load our training data `FashionMNIST` from `torchvision`, which has already been preprocessed into the scale of [0, 1]. We then wrap the dataset into an instance of `torch.utils.data.Dataloader`."
```

Reply:
i think the right side is open, the standardization is dividing by 256 from the original scale [0, 255].